media/js/base/stat-counter.js

```diff
@@ -0,0 +1,11 @@
+var STATCOUNTER_ID = document.getElementsByTagName('html')[0].getAttribute('data-statcounter-id');
```

Comment:
If the script needs to reference some global variables you can still make them available on window e.g `var sc_security = window.sc_security = 'foo'` (Adding a comment to explain this would also be good). This should still allow you to use an IIFE.